Library/Homebrew/os/mac/mach.rb

```diff
@@ -202,3 +199,7 @@ def dylib_id
     mach_metadata.dylib_id
   end
 end
+
+class Pathname
+  include(MachO)
+end
```

Comment:
Actually, I move this from `Library/Homebrew/extend/pathname.rb`. What I'm thinking is to put OS related code inside `os` folder.